packages/eslint-plugin-pf-codemods/src/rules/v6/emptyStateNonExportedComponents/emptyState-nonExported-components.test.ts

```diff
@@ -0,0 +1,121 @@
+        },
+      ],
+    },
+    {
+      code: `import { EmptyStateHeader as CustomHeader } from '@patternfly/react-core';`,
+      output: `import { EmptyStateHeader as CustomHeader } from '@patternfly/react-core';`,
+      errors: [
+        {
+          message: `EmptyStateHeader is no longer exported by PatternFly. This rule will not fix any imports, as our cleanup rule handles removal of unused imports.`,
+          type: "ImportSpecifier",
+        },
+      ],
+    },
+    {
+      code: `export { EmptyStateHeader } from '@patternfly/react-core';`,
+      output: ``,
+      errors: [
+        {
+          message: `EmptyStateHeader is no longer exported by PatternFly.`,
+          type: "ExportNamedDeclaration",
+        },
+      ],
+    },
+    {
+      code: `export { EmptyStateHeader as CustomHeader } from '@patternfly/react-core';`,
+      output: ``,
+      errors: [
+        {
+          message: `EmptyStateHeader is no longer exported by PatternFly.`,
+          type: "ExportNamedDeclaration",
+        },
+      ],
+    },
+    {
+      code: `export { EmptyStateHeader, EmptyStateIcon } from '@patternfly/react-core';`,
+      output: ``,
+      errors: [
+        {
+          message: `EmptyStateHeader and EmptyStateIcon are no longer exported by PatternFly.`,
+          type: "ExportNamedDeclaration",
+        },
+      ],
+    },
+    {
+      code: `export { EmptyStateHeader, Button, EmptyStateIcon } from '@patternfly/react-core';`,
+      output: `export {  Button } from '@patternfly/react-core';`,
+      errors: [
+        {
+          message: `EmptyStateHeader is no longer exported by PatternFly.`,
+          type: "ExportNamedDeclaration",
+        },
+        {
+          message: `EmptyStateIcon is no longer exported by PatternFly.`,
+          type: "ExportNamedDeclaration",
+        },
+      ],
+    },
+    {
+      code: `import { EmptyStateHeader } from '@patternfly/react-core';export { EmptyStateHeader };`,
+      output: `import { EmptyStateHeader } from '@patternfly/react-core';`,
+      errors: [
+        {
+          message: `EmptyStateHeader is no longer exported by PatternFly. This rule will not fix any imports, as our cleanup rule handles removal of unused imports.`,
+          type: "ImportSpecifier",
+        },
+        {
+          message: `EmptyStateHeader is no longer exported by PatternFly.`,
+          type: "ExportNamedDeclaration",
+        },
+      ],
+    },
+    {
+      code: `import { EmptyStateHeader as CustomHeader } from '@patternfly/react-core';export { CustomHeader };`,
+      output: `import { EmptyStateHeader as CustomHeader } from '@patternfly/react-core';`,
+      errors: [
+        {
+          message: `EmptyStateHeader is no longer exported by PatternFly. This rule will not fix any imports, as our cleanup rule handles removal of unused imports.`,
+          type: "ImportSpecifier",
+        },
+        {
+          message: `CustomHeader is no longer exported by PatternFly.`,
```

Comment:
I wonder if saying something like: `EmptyStateHeader (in your case CustomHeader) is no longer exported by PatternFly.` would be better, but it's probably not worth the time spent on the change. I guess there won't be many (if any) consumers reexporting it like this and it still is good enough info imo.